### PR TITLE
Create PpDataType

### PR DESCRIPTION
**Pull request by Jzow**
This Java class is used for datatype convert of paddlepadle engine

## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by Jzow** [comment]
Oh, sorry, I think my branch should come from djl:paddle , rather than djl:patch-1

File: paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package ai.djl.paddlepaddle.engine;
```
Comment (frankfliu) [inline paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java:1]: Please add License header to each java file.

File: paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+/**
+ * ClassName: PpDataType
+ * Description: Helper to convert between {@link DataType} an the PaddlePaddle internal DataTypes.
+ * Author: James Zow
+ * Date: 2020/11/14 15:29
+ * Version: 1.0
+ **/
+public class PpDataType {
```
Comment (frankfliu) [inline paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java:13]:
```suggestion
public final class PpDataType {
```

File: paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import ai.djl.ndarray.types.DataType;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+/**
+ * ClassName: PpDataType
+ * Description: Helper to convert between {@link DataType} an the PaddlePaddle internal DataTypes.
+ * Author: James Zow
+ * Date: 2020/11/14 15:29
+ * Version: 1.0
+ **/
```
Comment (frankfliu) [inline paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java:12]:
```suggestion
 
 **/
```

File: paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (frankfliu) [inline paddlepaddle/paddlepaddle-engine/src/main/java/ai/djl/paddlepaddle/engine/PpDataType.java:2]:
```suggestion
 * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
